imap/jmap_calendar.c

```diff
@@ -1276,6 +1277,17 @@ static void setcalendar_parseprops(jmap_req_t *req,
         props->share.overwrite_acl = 1;
         props->transp = -1;
         props->comp_types = -1;
+
+        /* Any externally-stored properties? */
+        const char *key;
+        json_t *val;
+        json_object_foreach(arg, key, val) {
+            const jmap_property_t *prop = jmap_property_find(key, calendar_props);
+            if (prop && (prop->flags & JMAP_PROP_EXTERNAL)) {
+                props->ext_props = 1;
```

Comment:
nit: here and below the `bool` value is set using numerics, rather than `true` and `false`. We don't have a guideline for this, but we might want to always use `true` and `false` to make the value type of the assigned variable clear. Also, in C23 they became keywords!